content/Startup.cs

```diff
@@ -97,7 +102,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             }
 #endif
 
-            app.UseResponseCompression();
```

Comment:
I had to disable UseResponseCompression because with it hot reload from Westwind.AspnetCore.LiveReload doesn’t work for some reason